<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pic_hard_syncClass</class>
 <widget class="QMainWindow" name="pic_hard_syncClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1429</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pic_hard_sync</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1421</width>
      <height>901</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>  RGB  图  像  显  示  </string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>461</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {     
border: 1px solid gray;     
border-radius: 9px;     
margin-top: 0.5em;
}

QGroupBox::title {     
subcontrol-origin: margin;     
left: 10px;     
padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string>深度鱼眼相机</string>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>201</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>1、灰度鱼眼相机_1</string>
       </property>
       <widget class="QLabel" name="label_gray_fisheye_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>201</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>2、灰度鱼眼相机_2</string>
       </property>
       <widget class="QLabel" name="label_gray_fisheye_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>461</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {     
border: 1px solid gray;     
border-radius: 9px;     
margin-top: 0.5em;
}

QGroupBox::title {     
subcontrol-origin: margin;     
left: 10px;     
padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string>RGB鱼眼相机</string>
      </property>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>201</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>3、RGB鱼眼相机_1</string>
       </property>
       <widget class="QLabel" name="label_RGB_fisheye_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>201</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>4、RGB鱼眼相机_2</string>
       </property>
       <widget class="QLabel" name="label_RGB_fisheye_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>1381</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {     
border: 1px solid gray;     
border-radius: 9px;     
margin-top: 0.5em;
}

QGroupBox::title {     
subcontrol-origin: margin;     
left: 10px;     
padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string>RGB相机</string>
      </property>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>201</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>5、RGB_1</string>
       </property>
       <widget class="QLabel" name="label_RGB_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>201</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>6、RGB_2</string>
       </property>
       <widget class="QLabel" name="label_RGB_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_10">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>30</y>
         <width>201</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>7、RGB_3</string>
       </property>
       <widget class="QLabel" name="label_RGB_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>30</y>
         <width>201</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>8、RGB_4</string>
       </property>
       <widget class="QLabel" name="label_RGB_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_12">
       <property name="geometry">
        <rect>
         <x>920</x>
         <y>30</y>
         <width>201</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>9、RGB_5</string>
       </property>
       <widget class="QLabel" name="label_RGB_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_13">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>30</y>
         <width>201</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>10、RGB_6</string>
       </property>
       <widget class="QLabel" name="label_RGB_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_14">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>250</y>
         <width>201</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>11、深度_1</string>
       </property>
       <widget class="QLabel" name="label_RGB_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_15">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>250</y>
         <width>201</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>12、深度_2</string>
       </property>
       <widget class="QLabel" name="label_RGB_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_16">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>250</y>
         <width>201</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>13、深度_3</string>
       </property>
       <widget class="QLabel" name="label_RGB_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_17">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>250</y>
         <width>201</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>14、深度_4</string>
       </property>
       <widget class="QLabel" name="label_RGB_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_18">
       <property name="geometry">
        <rect>
         <x>920</x>
         <y>250</y>
         <width>201</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>15、RGB_7</string>
       </property>
       <widget class="QLabel" name="label_RGB_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_19">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>250</y>
         <width>201</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>16、RGB_8</string>
       </property>
       <widget class="QLabel" name="label_RGB_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_20">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>10</y>
        <width>401</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {     
border: 1px solid gray;     
border-radius: 9px;     
margin-top: 0.5em;
}

QGroupBox::title {     
subcontrol-origin: margin;     
left: 10px;     
padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string>注册信息</string>
      </property>
      <widget class="QLabel" name="label_ID">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>用户ID：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ID">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_gestureID">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>90</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_gestureID">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>手势类别：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_gestureNO">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>手势序号：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_regist">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>190</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>注册用户</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_del">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>190</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>删除用户</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_status">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>230</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_gestureNO">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>140</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>  设  置  </string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_sample_frame">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>120</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>125</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_sample_frame">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>120</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>单次采样帧数：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_save_path">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>190</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>存储路径：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_select_path">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>250</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_root_path">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>200</y>
        <width>811</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>../data</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>320</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>40</number>
      </property>
      <property name="maximum">
       <number>160</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_iAeTarget">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>320</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>工业相机曝光亮度：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1429</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pic_hard_sync.qrc"/>
 </resources>
 <connections/>
</ui>
